apps/files/lib/Controller/ViewController.php

```diff
@@ -266,7 +266,7 @@ public function index($dir = '', $view = '', $fileid = null, $fileNotFound = fal
 		$nav->assign('quota', $storageInfo['quota']);
 		$nav->assign('usage_relative', $storageInfo['relative']);
 
-		$nav->assign('webdav_url', \OCP\Util::linkToRemote('dav/files/' . $user));
+		$nav->assign('webdav_url', \OCP\Util::linkToRemote('dav/files/' . urlencode($user)));
```

Comment:
```suggestion
		$nav->assign('webdav_url', \OCP\Util::linkToRemote('dav/files/' . rawurlencode($user)));
```

I think? Always forget which one was better

Reply:
I think rawurlencode:

[If you are encoding path segment, use rawurlencode().
If you are encoding query component, use urlencode().](https://www.gyrocode.com/articles/php-urlencode-vs-rawurlencode/)